### ISSUE TITLE
[Bug] Logged in to check my farms and Chia no longer running. When I restart it is prompting me for the 24 key passphrase again?

### ISSUE THREAD

**Issue by OldButtKicking** [Chia-Network/chia-blockchain#10297]
### What happened?

Who knows but Chia is no longer running but was when I left it several days ago? It appears to have corrupted my wallet when it died. There are other forks running that have not had any issue. If anyone could acknowledge it is worth my time logging these issues please let me know because my experience to date is ZERO response for anything I report.

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
looks like the logs are full of
2022-02-16T16:38:49.896 farmer chia.rpc.rpc_server        : WARNING  Cannot connect to daemon at ws://localhost:55400
2022-02-16T16:38:51.115 wallet chia.rpc.rpc_server        : WARNING  Cannot connect to daemon at ws://localhost:55400
2022-02-16T16:38:54.379 full_node chia.rpc.rpc_server     : WARNING  Cannot connect to daemon at ws://localhost:55400
2022-02-16T16:38:55.961 farmer chia.rpc.rpc_server        : WARNING  Cannot connect to daemon at ws://localhost:55400
for days :(

I think the UI must have crashed here but it could have been sooner. The network has not disconnected and their are a number of forks running that do not seem to have had any issues.


2022-02-18T01:37:28.780 wallet chia.rpc.rpc_server        : WARNING  Cannot connect to daemon at ws://localhost:55400
2022-02-18T01:37:31.100 full_node chia.rpc.rpc_server     : WARNING  Cannot connect to daemon at ws://localhost:55400
2022-02-18T01:37:32.007 daemon __main__                   : INFO     Starting Daemon Server
2022-02-18T01:37:32.008 daemon __main__                   : INFO     Not implemented
2022-02-18T01:37:32.013 daemon __main__                   : INFO     Waiting Daemon WebSocketServer closure
2022-02-18T01:37:32.307 daemon __main__                   : INFO     Register service {'service': 'chia_wallet'}
2022-02-18T01:37:32.308 daemon __main__                   : INFO     registered for service chia_wallet
2022-02-18T01:37:32.308 daemon __main__                   : INFO     {'success': True}
2022-02-18T01:37:32.354 daemon __main__                   : INFO     Register service {'service': 'chia_farmer'}
2022-02-18T01:37:32.354 daemon __main__                   : INFO     registered for service chia_farmer
2022-02-18T01:37:32.355 daemon __main__                   : INFO     {'success': True}
2022-02-18T01:37:32.989 daemon __main__                   : INFO     Register service {'service': 'wallet_ui'}
2022-02-18T01:37:32.989 daemon __main__                   : INFO     registered for service wallet_ui
2022-02-18T01:37:32.990 daemon __main__                   : INFO     {'success': True}
2022-02-18T01:37:32.990 daemon __main__                   : INFO     Register service {'service': 'chia_plotter'}
2022-02-18T01:37:32.991 daemon __main__                   : INFO     registered for service chia_plotter
2022-02-18T01:37:32.991 daemon __main__                   : INFO     {'success': True, 'service': 'chia_plotter', 'queue': []}
2022-02-18T01:37:33.112 daemon __main__                   : INFO     Register service {'service': 'chia_full_node'}
2022-02-18T01:37:33.113 daemon __main__                   : INFO     registered for service chia_full_node
2022-02-18T01:37:33.113 daemon __main__                   : INFO     {'success': True}
2022-02-18T01:37:33.944 full_node chia.full_node.full_node: INFO       Finished signage point 38/64: CC: 90834630c817a232ed5b0751ded3220db64e60060f4e1e77358a9bd1359d64f7 RC: 876d824c48a9ac0eb893df747e3be8c4c7534561e4b6fb6d382d8423da625a19 
2022-02-18T01:37:42.702 full_node chia.full_node.full_node: INFO       Finished signage point 39/64: CC: 2a63b2c008ca6810532a5e4610cdb390f505d44422638dd9afeee80483c9b912 RC: 9f3ece387d700ee5603bdcd26cc5a9091fc9f448b3da3a0df631b17cfe025819 
2022-02-18T01:37:44.493 full_node chia.full_node.full_node: INFO     Added unfinished_block d753e9bd180b581a4d72536af2bfe71798e2a30303ab95c95623724c84b1b407, not farmed by us, SP: 39 farmer response time: 1.7790, Pool pk xch1f0ryxk6qn096hefcwrdwpuph2hm24w69jnzezhkfswk0z2jar7aq5zzpfj, validation time: 0.0120 seconds, cost: 0, percent full: 0.0%
2022-02-18T01:37:48.997 daemon __main__                   : INFO     Running check_keys now that the keyring is unlocked
2022-02-18T01:37:49.306 wallet asyncio                    : ERROR    Task exception was never retrieved
future: <Task finished name='Task-1290694' coro=<WebSocketCommonProtocol.send() done, defined at websockets\protocol.py:521> exception=ConnectionClosedError('code = 1006 (connection closed abnormally [internal]), no reason')>
Traceback (most recent call last):
  File "asyncio\windows_events.py", line 457, in finish_recv
OSError: [WinError 64] The specified network name is no longer available

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "websockets\protocol.py", line 827, in transfer_data
  File "websockets\protocol.py", line 895, in read_message
  File "websockets\protocol.py", line 971, in read_data_frame
  File "websockets\protocol.py", line 1047, in read_frame
  File "websockets\framing.py", line 105, in read
  File "asyncio\streams.py", line 723, in readexactly
  File "asyncio\streams.py", line 517, in _wait_for_data
  File "asyncio\proactor_events.py", line 280, in _loop_reading
  File "asyncio\windows_events.py", line 812, in _poll
  File "asyncio\windows_events.py", line 461, in finish_recv
ConnectionResetError: [WinError 64] The specified network name is no longer available

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "websockets\protocol.py", line 555, in send
  File "websockets\protocol.py", line 803, in ensure_open
websockets.exceptions.ConnectionClosedError: code = 1006 (connection closed abnormally [internal]), no reason
2022-02-18T01:37:49.323 farmer asyncio                    : ERROR    Task exception was never retrieved
future: <Task finished name='Task-365965' coro=<WebSocketCommonProtocol.send() done, defined at websockets\protocol.py:521> exception=ConnectionClosedError('code = 1006 (connection closed abnormally [internal]), no reason')>
Traceback (most recent call last):
  File "asyncio\windows_events.py", line 457, in finish_recv
OSError: [WinError 64] The specified network name is no longer available

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "websockets\protocol.py", line 827, in transfer_data
  File "websockets\protocol.py", line 895, in read_message
  File "websockets\protocol.py", line 971, in read_data_frame
  File "websockets\protocol.py", line 1047, in read_frame
  File "websockets\framing.py", line 105, in read
  File "asyncio\streams.py", line 723, in readexactly
  File "asyncio\streams.py", line 517, in _wait_for_data
  File "asyncio\proactor_events.py", line 280, in _loop_reading
  File "asyncio\windows_events.py", line 812, in _poll
  File "asyncio\windows_events.py", line 461, in finish_recv
ConnectionResetError: [WinError 64] The specified network name is no longer available

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "websockets\protocol.py", line 555, in send
  File "websockets\protocol.py", line 803, in ensure_open
websockets.exceptions.ConnectionClosedError: code = 1006 (connection closed abnormally [internal]), no reason
2
```

**Comment by OldButtKicking** [comment]
After trying to provide the 24 keys all i get is non type error and fingerprint error. Pressing ok just takes me from one to the other.

**Comment by OldButtKicking** [comment]
had to kill Chia to get passed it

**Comment by OldButtKicking** [comment]
will come back tomorrow to see if i can fix it or someone can offer some advice.

**Comment by esaung** [comment]
did it come back for you?

**Comment by OldButtKicking** [comment]
Yes but I did not re-enter my keys. I realized when I came back the next day there were several zombie processes running for Chia. Even though the UI appeared to close down correctly. I terminated those through task manager and when I restarted Chia it recognized my wallet again and the keys it contained. I suspect the wallet was one of the zombie processes and when I tried to restart it could not open the wallet as it was already open. This is why it prompted me again to create a new wallet or enter my existing keys. I'm sure rebooting would have also fixed it but I didn't want to reboot as I had a bunch of forks running.
